light_curves/scale_up.md

```diff
@@ -0,0 +1,559 @@
+---
+jupytext:
+  cell_metadata_filter: -all
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Make Multi-Wavelength Light Curves for Large Samples
+
++++
+
+## Learning Goals
+
+By the end of this tutorial, you will be able to:
+
+- Parallelize the code demonstrated in the [light_curve_generator](light_curve_generator.md) notebook to get multi-wavelength light curves.
+- Launch a run using a large sample of objects, monitor the run's progress automatically, and understand its resource usage (CPU and RAM).
+- Understand some general challenges and requirements when scaling up code.
+
++++
+
+## Introduction
+
++++
+
+This notebook shows how to collect multi-wavelength light curves for a large sample of target objects.
+This is a scaling-up of the [light_curve_generator](light_curve_generator.md) and assumes you are familiar with the content of that notebook.
+This code should be used for sample sizes greater than a few hundred.
+Many of the challenges, needs, and wants discussed in the appendix and addressed throughout the notebook are common when scaling up code and the ideas can be applied to other use cases.
+
+Notebook sections are:
+
+- "Overview" describes functionality of the included bash script and python helper functions. Compares some parallel processing options.
+- "Example 1" shows how to launch a large-scale run using the bash script, monitor its progress automatically, and diagnose a problem (out of RAM).
```

Comment:
I agree and have tried but am also unable to get it to work. I tried the only option from the [myst-parser docs](https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html) that seemed like it should work in all cases (Annotating a syntax block with `(target)=`). External links to files or URLs seem to work, but not links to a section within the notebook. I'm guessing this doesn't work because the markdown in different notebook cells is isolated from the others in some way.

Interested to hear if @bsipocz knows of a solution.